src/confluent_kafka/src/Admin.c

```diff
@@ -1531,6 +1528,365 @@ const char Admin_list_consumer_groups_doc[] = PyDoc_STR(
+
+        PyObject *password;
+        const unsigned char *password_c;
+        Py_ssize_t password_c_size;
+
+        PyObject *scram_credential_info;
+        PyObject *mechanism;
+        int32_t iterations;
+        int mechanism_val;
+
+        if (!PyArg_ParseTupleAndKeywords(args, kwargs, "OO|f", kws,
+                                         &alterations, &future,
+                                         &options.request_timeout))
+                return NULL;
+
+        if (!PyList_Check(alterations)) {
+                PyErr_SetString(PyExc_TypeError,
+                                "Expected non-empty list of Alteration "
+                                "objects");
+                return NULL;
+        }
+        UserScramCredentialAlteration_type = cfl_PyObject_lookup("confluent_kafka.admin",
+                                                  "UserScramCredentialAlteration");
+
+
+        if (!UserScramCredentialAlteration_type) {
+                        PyErr_SetString(PyExc_ImportError,
+                                "Not able to load UserScramCredentialAlteration type");
+                        goto err;
+        }
+
+        UserScramCredentialUpsertion_type = cfl_PyObject_lookup("confluent_kafka.admin",
+                                                  "UserScramCredentialUpsertion");
+        if (!UserScramCredentialUpsertion_type) {
+                        PyErr_SetString(PyExc_ImportError,
+                                "Not able to load UserScramCredentialUpsertion type");
+                        goto err;
+        }
+        UserScramCredentialDeletion_type = cfl_PyObject_lookup("confluent_kafka.admin",
+                                                  "UserScramCredentialDeletion");
+        if (!UserScramCredentialDeletion_type) {
+                        PyErr_SetString(PyExc_ImportError,
+                                "Not able to load UserScramCredentialDeletion type");
+                        goto err;
+        }
+
+        ScramCredentialInfo_type = cfl_PyObject_lookup("confluent_kafka.admin",
+                                                  "ScramCredentialInfo");
+        if (!ScramCredentialInfo_type) {
+                        PyErr_SetString(PyExc_ImportError,
+                                "Not able to load ScramCredentialInfo type");
+                        goto err;
+        }
+
+        ScramMechanism_type = cfl_PyObject_lookup("confluent_kafka.admin",
+                                                  "ScramMechanism");\
+        if (!ScramMechanism_type) {
+                        PyErr_SetString(PyExc_ImportError,
+                                "Not able to load ScramMechanism type");
+                        goto err;
+        }
+
+        c_options = Admin_options_to_c(self, RD_KAFKA_ADMIN_OP_ALTERUSERSCRAMCREDENTIALS,
+                                       &options, future);
+        if (!c_options)
+                return NULL; /* Exception raised by options_to_c() */
```

Comment:
I think we should use `goto err` here instead of `return NULL` as many objects are created before this.